beta/src/content/apis/react/useTransition.md

```diff
@@ -2,19 +2,1393 @@
 title: useTransition
 ---
 
-<Wip>
+<Intro>
 
-This section is incomplete, please see the old docs for [useTransition.](https://reactjs.org/docs/hooks-reference.html#usetransition)
+`useTransition` is a React Hook that lets you mark a state transition as non-urgent.
 
-</Wip>
+```js
+const [isPending, startTransition] = useTransition()
```

Comment:
I'm sure this is intentional and it's been covered in the past, but can you talk about the decision on whether or not to include a semicolon in these intro code blocks?

On one hand, when it's a statement (and especially a jsx tag), a semicolon seems awkward or distracting. On the other, especially when it's an assignment, it sort of seems incomplete or like it could be a mistake.

Reply:
I don't care strongly either way, the idea was just that these intro blocks aren't always complete statements and they're more about showing the signature. E.g. some aren't even valid JS syntax, like using `?` for optional args. If someone feels like these should have semis I'm happy to take a PR adding them.